src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -402,7 +402,7 @@ public void Deconstruct(out int a, out string b, out string c)
             comp.VerifyDiagnostics();
         }
 
-        [Fact(Skip = "PROTOTYPE(tuples)")]
+        [Fact]
```

Comment:
I'd add a test mixing deconstruction assignment and a regular ones. Just to make sure it works.
``` (a, b) = c = d = e; ```
and mix with postfix operator. - Just to make sure deconstruction composes in the few scenarios where it is possible.
``` (a, b) = c++; ```    May need to declare extension Deconstruct on Int32 to make this work.